wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+            help: "Setting this means Motif Discovery and Enrichment analysis will be performed.",
+            example: true,
+        }
+    }
+
+    input {
+        # group: reference_genome
+        File reference
+        File? spikein_reference
+        File? blacklist
+        File gtf
+        Array[File]? bowtie_index
+        Array[File]? spikein_bowtie_index
+        Array[File]? motif_databases
+
+        # group: input_genomic_data
+        Array[String]? sample_sraid
+        Array[File]? sample_fastq
+
+        # group: analysis_parameter
+        String? results_name
+        Boolean run_motifs = true
+
+    }
+
+    String pipeline_ver = "v2.0.0"
+
+    ### ---------------------------------------- ###
+    ### ------------ S E C T I O N 1 ----------- ###
+    ### ------ Pre-process Analysis Files ------ ###
+    ### ---------------------------------------- ###
+
+    # Process SRRs
+    if (defined(sample_sraid)) {
+        # Download sample file(s) from SRA database
+        # outputs:
+        #    fastqdump.fastqfile : downloaded sample files in fastq.gz format
+        Array[String] string_sra = [
+            1,
+        ]  #buffer to allow for sra_id optionality
+        Array[String] s_sraid = select_first([
+            sample_sraid,
+            string_sra,
+        ])
+        scatter (eachsra in s_sraid) {
+            call sra.fastqdump { input:
+                sra_id = eachsra,
+                cloud = false,
+            }
+        }  # end scatter each sra
+
+        Array[File] sample_srafile = flatten(fastqdump.fastqfile)
+    }  # end if sample_sraid
+
+    # Generating INDEX files
+    #1. Bowtie INDEX files if not provided
+    if (!defined(bowtie_index)) {
+        # create bowtie index when not provided
+        call bowtie.index as bowtie_idx { input: reference = reference }
+    }
+    #2. Make sure indexes are six else build indexes
+    if (defined(bowtie_index)) {
+        # check total number of bowtie indexes provided
+        Array[String] string_bowtie_index = [
+            1,
+        ]  #buffer to allow for bowtie_index optionality
+        Array[File] int_bowtie_index = select_first([
+            bowtie_index,
+            string_bowtie_index,
+        ])
+        if (length(int_bowtie_index) != 6) {
+            # create bowtie index if 6 index files aren't provided
+            call bowtie.index as bowtie_idx_2 { input: reference = reference }
+        }
+    }
+    Array[File] actual_bowtie_index = select_first([
+        bowtie_idx_2.bowtie_indexes,
+        bowtie_idx.bowtie_indexes,
+        bowtie_index,
+    ])
+
+    # Spike-in DNA
+    #3. Bowtie INDEX files if not provided
+    String string_spikein = "1"
+    Array[String] string_spikein_buffer = [
+        1,
+    ]
+    if (!defined(spikein_bowtie_index) && defined(spikein_reference)) {
+        # create bowtie index on spikein genome
+        call bowtie.index as spikein_bowtie_idx { input: reference = select_first([
```

Comment:
I think `reference` should get bumped to a new line since its RHS is line split.

Reply:
Agree